Gemfile.lock

```diff
@@ -762,7 +762,7 @@ DEPENDENCIES
   wkhtmltopdf-binary
 
 RUBY VERSION
-   ruby 2.1.5p273
+  ruby 2.1.5p273
 
 BUNDLED WITH
-   1.15.1
+  1.15.1
```

Comment:
@enricostano I did modify it myself (to restore the file). I'm actually a bit in the dark as to why the file was altered in the first place; as far as I know, I didn't manually edit it. I didn't realize the change had been made until I made my commit.

I'm unfamiliar with Bundler; does it dynamically write the `Gemfile.lock` itself each time Bundler is run?